justfile

```diff
@@ -10,7 +10,7 @@ docs:
 
 # Do a dev install.
 dev:
-  pip install -e '.[dev]'
+  pip install -e '.[dev, track]'
```

Comment:
This is not needed for baseline development? 

I think you want a second option that installs track if you want it in the development